### PR TITLE
Fixes syndicate saboteurs and lone infiltrators

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
This PR allows admins to create non-progtots again through one-click antag, removes saboteurs (traitors default to non-prog now), and fixes lone infil **again** (thank you regressions) since I had to touch things relating to it

## How This Contributes To The Skyrat Roleplay Experience

Bugs are bad, I've been begged to fix the create antag thing for a while
## Proof of Testing

<details><summary>screenshots</summary

image

image

  
</details>

## Changelog

:cl:
spellcheck: Syndicate saboteur is now traitor again
fix: One-click-antag non-progtots now work again
fix: Lone infiltrator spawns again
/:cl:

### CODE REVIEW

**Review by tf-4 — APPROVED** [review]

**Comment by Zergspower** [comment]
time to get my ERP antag game on again (I am cursed here where the minute the pants go down that window pops up)

File: modular_skyrat/modules/Midroundtraitor/code/datum_traitor.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	var/infil_outfit = /datum/outfit/syndicateinfiltrator
 	preview_outfit = /datum/outfit/lone_infiltrator_preview
 	job_rank = ROLE_LONE_INFILTRATOR
+	progression_enabled = TRUE
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/Midroundtraitor/code/datum_traitor.dm]: And why did the infiltrator have prog-tot objectives, again? Wasn't the whole point literally not to have prog-tots anymore?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/Midroundtraitor/code/datum_traitor.dm (hunk 1 above)
Comment (Zonespace27) [reply modular_skyrat/modules/Midroundtraitor/code/datum_traitor.dm]: *(discussed and resolved on-discord, an oversight from the original progtot PR)*

**Review by GoldenAlpharex — APPROVED** [review]
Should be good now!

**Pull request merged**